api/app/Console/Commands/DepartmentDeduplication.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Department;
+use App\Models\Pool;
+use App\Models\PoolCandidate;
+use App\Models\PoolCandidateSearchRequest;
+use App\Models\TalentNomination;
+use App\Models\User;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+
+class DepartmentDeduplication extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:department-deduplication';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Following a spreadsheet map, update references to duplicates then delete the duplicates';
+
+    /**
+     * Tables to reference, computed via SQL command
+     * https://dba.stackexchange.com/questions/265732/postgresql-how-to-list-all-foreign-key-link-to-primary-key-of-a-table
+     *
+     * pools	department_id
+     * department_user_next_role	department_id
+     * pool_candidate_search_requests	department_id
+     * pool_candidates	placed_department_id
+     * team_department	department_id
+     * users	computed_department
+     * department_user_career_objective	department_id
+     * talent_nominations	nominator_fallback_department_id
+     * talent_nominations	advancement_reference_fallback_department_id
+     */
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        // map created from spreadsheet
+        $departmentMap = [
+            [
+                'name_en' => 'Accessibility Standards Canada',
+                'keep_number' => 903,
+                'remove_number' => 199,
+            ],
+            [
+                'name_en' => 'Housing, Infrastructure and Communities Canada',
+                'keep_number' => 201,
+                'remove_number' => 142,
+            ],
+            [
+                'name_en' => 'Innovation, Science and Economic Development Canada',
+                'keep_number' => 33,
+                'remove_number' => 207,
+            ],
+            [
+                'name_en' => 'Polar Knowledge Canada',
```

Comment:
as per the spreadsheet I believe this should be modified to `Canadian High Arctic Research Station`